sdk/go/common/util/cmdutil/exit.go

```diff
@@ -160,18 +161,38 @@ func exitErrorCodef(code int, format string, args ...interface{}) {
 	os.Exit(code)
 }
 
+type stdMultiError interface {
```

Comment:
Is this really the "correct" way to detect an errors.Join error?

Reply:
Yeah. They didn't export an interface or a function to break apart a joined error.
They only documented the contract that `Unwrap() []error` = joined error of any kind (including `fmt.Errorf` with multiple `%w`s).

Reply:
```
switch x := err.(type) {
		case interface{ Unwrap() []error }:
```

That's maybe a neater way to do it here as well rather than adding a new name to the package.

Reply:
Yeah, I prefer the inline interface too. I may have gone with the named one for documentation purposes. A comment will suffice for the inline one, though. I'll switch to it.